retrieval/target.go

```diff
@@ -359,9 +364,15 @@ func (t *Target) scrape(sampleAppender storage.SampleAppender) (err error) {
 	}()
 
 	for samples := range t.ingestedSamples {
-		for _, s := range samples {
-			s.Metric.MergeFromLabelSet(baseLabels, clientmodel.ExporterLabelPrefix)
-			sampleAppender.Append(s)
+		if t.proxy {
+			for _, s := range samples {
+				sampleAppender.Append(s)
```

Comment:
@brian-brazil The question is once more how to handle an empty label internally. See that other issue.

Scenario 1:
So PGW attaches `instance=""`, Prometheus with `honor_labels` then leaves that label intact.
Question: What's the implication for saving the metric and querying it?
@fabxc is playing with that right now, @juliusv might know.

Scenario 2:
PGW attaches `instance="`. Prometheus server with `honor_labels` then removes that label, so that no instance label at all is saved. Then the same Prometheus server is exporting the metric to a parent server via federation. The parent server _will_ attach an `instance` label with the host and port of the child prometheus server then, won't it? That's a problem, right?